readme.md

```diff
@@ -25,7 +40,9 @@ Your Answer:
 Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
-```text
+```
+
+Ajax request data from an API. When you make a request, you need to be specific with regards to what data you are requesting. In the example above, We requested JSON data from "kittengifs.com" using a "get" method.
 
 ```
```

Comment:
:+1:

Also looking for instructions on (1) how we would pull information from a form (e.g., `$( "input" ).val( )`) and (2) how we would trigger that AJAX call in the first place (e.g., event listener).